.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,26 @@ assignees: ''
 
 ---
 
-<!-- Thanks for helping us to improve k3s! We welcome all bug reports. Please fill out each area of the template so we can better help you.  ***You can delete this message portion of the bug report.*** -->
+<!-- Thanks for helping us to improve K3s! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
 
-**Version:**
-<!-- Provide the output from `k3s -v` -->
+**Version Info:**
+<!-- Provide the output from "k3s -v" -->
```

Comment:
```suggestion
<!-- Provide the output from "K3s -v" -->
```

Reply:
binary is lowercase OOTB, we should not capitalize